src/lib/logical_query_plan/abstract_lqp_node.cpp

```diff
@@ -218,9 +218,9 @@ bool AbstractLQPNode::shallow_equals(const AbstractLQPNode& rhs, const LQPNodeMa
   return _on_shallow_equals(rhs, node_mapping);
 }
 
-const std::vector<std::shared_ptr<AbstractExpression>>& AbstractLQPNode::column_expressions() const {
+std::vector<std::shared_ptr<AbstractExpression>> AbstractLQPNode::column_expressions() const {
   Assert(left_input() && !right_input(),
-         "Can only forward input expressions iff there is a left input and no right input");
+         "Can only forward input expressions if there is a left input and no right input");
```

Comment:
Not a typo: https://en.wiktionary.org/wiki/iff

Feel free to write it out

Reply:
Good hint, thanks!